app/styles/vars.scss

```diff
@@ -1,5 +1,5 @@
 $page-top-margin: 14px;
-$event-lead-image-height: 500px;
+$event-lead-image-height: 400px;
```

Comment:
this change will impact other areas too. please add the screenshot of those areas where it is used.